fdbcli/linenoise/linenoise.c

```diff
@@ -407,6 +410,18 @@ void linenoiseSetCompletionCallback(linenoiseCompletionCallback *fn) {
     completionCallback = fn;
 }
 
+/* Register a hits function to be called to show hits to the user at the
```

Comment:
`s/hits/hints/g`

Reply:
This is an exact copy paste of upstream code, it's not something that I wrote myself, and I'd prefer to keep this file identical to upstream.